packages/next/telemetry/events/version.ts

```diff
@@ -1,33 +1,85 @@
 const EVENT_VERSION = 'NEXT_CLI_SESSION_STARTED'
+const pageExtensions = ['tsx', 'ts', 'jsx', 'js']
```

Comment:
Why is data being duplicated here? This doesn't seem good. We need a place for our defaults.